isis/src/base/apps/nocam2map/nocam2map.h

```diff
@@ -1,10 +1,13 @@
-#ifndef nocam2map_h
-#define nocam2map_h
+#ifndef NoCam2Map_h
+#define NoCam2Map_h
```

Comment:
So far we have made these all lowercase, may want to continue following that trend